installers/metrics/ansible/roles/pgo-metrics/tasks/prometheus.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Deploy Prometheus
+  block:
+    - name: Set Prometheus Output Directory Fact
+      set_fact:
+        prom_output_dir: "{{ metrics_dir }}/output/prom"
+      tags: always
```

Comment:
Since these `tags: always` are in a block with `tags: install, update` will they be run for an `uninstall`? Should this always tag be here?

Reply:
Removed `tags: always` for the directory tasks in both `prometheus.yml` and `grafana.yml`.